### PR TITLE
Make progress indicator more consistent

### PR DESCRIPTION
**Pull request by hyamanieu**
Fixes https://github.com/holoviz/panel/issues/2685

### CODE REVIEW

**Comment by hyamanieu** [comment]
Wrong issue mentioned in my commit, it actually fixes #2196

File: panel/models/progress.ts [hunk 1]
```diff
@@ -49,8 +49,15 @@ export class ProgressView extends HTMLBoxView {
   }
 
   setValue(): void {
-    if (this.model.value != null)
+    console.log(this.model.value)
+    console.log(this.progressEl.value)
```
Comment (philippjfr) [inline panel/models/progress.ts:53]: Can you remove the log statements?

File: panel/widgets/indicators.py [hunk 2]
```diff
@@ -786,13 +796,8 @@ def __init__(self, **params):
 
         self.param.watch(self._update_layout, list(Viewable.param))
 
-        if self.value == 0:
-            # Hack: to give progress the initial look
```
Comment (philippjfr) [inline panel/widgets/indicators.py:790]: Definitely didn't like this hack but is this condition handled, i.e. under what configuration do you get an indeterminate bar and under what condition do you get an empty bar now?

Comment (hyamanieu) [reply panel/widgets/indicators.py:790]: In the current state, Tqdm cannot have a value of None, so no indeterminate bar, nor -1, so no empty bar (explaining the trick).
I am proposing in my last commit to simply copy the params `value` and `max` from Progress and set default to 0 for value. So now Tqdm can have an indeterminate state and no hack is programmed.

File: panel/models/progress.ts (hunk 1 above)
Comment (hyamanieu) [reply panel/models/progress.ts:53]: Removed in last commit

File: panel/widgets/indicators.py (hunk 2 above)
Comment (hyamanieu) [reply panel/widgets/indicators.py:790]: To repond to your question: now Tqdm can be indeterminate with a value of -1 (not happening in the gallery examples) and be empty with a value of 0.

Comment (philippjfr) [reply panel/widgets/indicators.py:790]: Okay, will check out the PR and play around. The Progress indicator can be used independently of Tqdm so I want to make sure that still works as expected.

**Comment by philippjfr** [comment]
Thanks @hyamanieu, appreciate it!

**Pull request merged**